setup.py

```diff
@@ -26,7 +26,7 @@ def run(self):
 
 setup(
     name='AxonDeepSeg',
-    python_requires='>=3.7, <3.8',
+    python_requires='>=3.9, <3.10',
```

Comment:
If I'm not mistaken, `ivadomed` is currently compatible with python 3.6, 3.7 and 3.8. And there is a [PR](https://github.com/ivadomed/ivadomed/pull/819) open to also be compatible with 3.9 but it is not done yet. Is that required to be able to use the ARM M1?

Reply:
Ok I wasn't sure - thanks for letting me know! Let me try with Python 3.8 now; if it worked with Python 3.9, I don't forsee any issue.

Reply:
Done in f834a1c and 6deeab9.